webui/react/Makefile

```diff
@@ -1,5 +1,10 @@
+swagger_client = "src/services/swagger"
+PROJECT_ROOT := "../../"
+
 .PHONY: get-deps
 get-deps:
+	# WARN this module also depends on the swagger generated api client
+	# which is not built or updated here
```

Comment:
question: do we want to get rid of this/improve this in future work, or is it just a caution for other people in this code? if the former, can we make a ticket/link it; if the latter, can we be a bit more clear on where one should go look if they want more information?

Reply:
hmm I don't see a good way of getting rid of this without more changes to our build system to introducing separate make targets and using different ones in CI vs isolated WebUI development vs local cluster build. I can add more information

Reply:
added wording to point the user to the target module. the info here would/could change if we make changes to the CI. I designed it this way to break up CI dependencies. if we don't want to do the process would be simpler but slower.